library/weaviate.py

```diff
@@ -205,6 +205,23 @@ def get_thread_by_id(self, thread_id: str):
         if len(results.objects)>0:
             return results.objects[0].properties
         return None
+
+    def get_email_metadata(self, email_id: str):
+        results = self.collection(WeaviateSchemas.EMAIL).query.fetch_objects(
+            filters=Filter.by_property("email_id").equal(email_id),
+        )
+        if len(results.objects)>0:
+            return [obj.properties for obj in results.objects]
+        return None
```

Comment:
If this method is meant to return a list, it's generally considered dangerous to return a null value for it when it's empty. See https://stackoverflow.com/questions/36779498/why-is-it-considered-good-practice-to-return-an-empty-collection

Reply:
In fact, whenever possible, I'd say it's best to avoid using a null response. You avoid a huge percentage of the bugs in software by following that at a high level. See

https://www.infoq.com/presentations/Null-References-The-Billion-Dollar-Mistake-Tony-Hoare/